examples/inference-deployments/mpt/mpt_7b_ft_handler.py

```diff
@@ -140,25 +142,21 @@ def __init__(self,
                                                        trust_remote_code=True)
         print('FT initialization complete')
 
-    def _parse_single_input(
-            self, inputs: Dict[str, Any]) -> Tuple[List[str], Dict[str, Any]]:
-        """Splits request into input strings and kwargs."""
-        if self.INPUT_STRINGS_KEY not in inputs or not isinstance(
-                inputs[self.INPUT_STRINGS_KEY], list):
+    def _parse_model_request(
+            self, model_request: Dict[str, Any]) -> Tuple[str, Dict[str, Any]]:
+        if self.INPUT_KEY not in model_request:
             raise RuntimeError(
-                'Input strings must be provided as a list to generate call')
+                f'{self.INPUT_KEY} must be provided to generate call')
```

Comment:
nit: add quotes so it's more obvious this is a literal key: `f'"{self.INPUT_KEY}" ...'`.